RobotCode/snobot2018/trajectory_config/CenterToLeftSwitch.yml

```diff
@@ -0,0 +1,35 @@
+config: !!com.team254.lib.trajectory.gen.TrajectoryGenerator$Config
```

Comment:
This is an example of the configuration file used to generate a full trajectory.  Has the movement parameters, and the waypoints to hit.  The standalone CoordinateGui can use this to generate the big csv files (not) shown above